From 5ebf29943b291110a816a418cb9d311d6172ac1e Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Sat, 22 Jul 2017 03:17:02 +0000
Subject: [PATCH] HEAD-2364 dhcpd zone runs aground in default RSS cap Reviewed
 by: Trent Mick <trentm@gmail.com> Reviewed by: Patrick Mooney
 <patrick.mooney@joyent.com> Approved by: Trent Mick <trentm@gmail.com>

---
 config/sapi/services/dhcpd/service.json | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/sapi/services/dhcpd/service.json b/config/sapi/services/dhcpd/service.json
index f236d234..0e9a300b 100644
--- a/config/sapi/services/dhcpd/service.json
+++ b/config/sapi/services/dhcpd/service.json
@@ -2,7 +2,7 @@
     "name": "dhcpd",
     "params": {
         "archive_on_delete": true,
-        "package_name": "sdc_128",
+        "package_name": "sdc_512",
         "image_uuid": "IMAGE_UUID",
         "maintain_resolvers": true,
         "networks": [ "admin" ],
-- 
2.21.0

